frontend/public/components/storage-class-form.tsx

```diff
@@ -979,6 +997,26 @@ export class StorageClassForm_ extends React.Component<
               claim is deleted. Defaults to &lsquo;Delete&rsquo;
             </span>
           </div>
+          <div className="form-group">
+            <label className="control-label co-required" htmlFor="expansion-switch">
+              Allow Expansion
+            </label>
+            <div>
+              <Switch
```

Comment:
I think size of the switch can be increased a bit.

Reply:
ack